CONFIGURATION.md

```diff
@@ -0,0 +1,311 @@
+
+-   `setNodeMinReadmitPeriod` - Set minimum node readmit period
+    When a node fails to respond or encounters an error while processing a request, the SDK will temporarily remove that node from the pool of available nodes. This is done to prevent the SDK from repeatedly sending requests to a node that is experiencing issues, which could further exacerbate the problem.
+    The `setNodeMinReadmitPeriod` method allows you to configure the minimum amount of time that a node must wait before it can be readmitted to the pool of available nodes. During this period, the SDK will not send any requests to that node, giving it time to recover or resolve any issues it may be experiencing.
+
+-   `setNodeMaxReadmitPeriod` - Set maximum node readmit period.
+    The setNodeMaxReadmitPeriod method allows you to configure the maximum amount of time that a node can be excluded from the pool of available nodes. After this period has elapsed, the SDK will automatically readmit the node to the pool, regardless of whether it has recovered or not.
+
+-   `setNodeWaitTime` - The `setNodeWaitTime` method in the Hedera JavaScript SDK is used to set the minimum amount of time (in milliseconds) that the SDK will wait before attempting to send a request to a node that has recently failed or encountered an error.
+
+```javascript
+const client = Client.forNetwork();
+client.setNodeWaitTime(5000); // Set node wait time to 5 seconds (5000 milliseconds)
+```
+
+### Network Update Settings
+
+-   `setNetworkUpdatePeriod` - The setNetworkUpdatePeriod method in the Hedera JavaScript SDK is used to configure the frequency at which the SDK updates its internal representation of the Hedera network topology.
+    The Hedera network is a distributed ledger system that consists of multiple nodes spread across different geographic locations. These nodes can join or leave the network at any time, and their availability and performance can vary depending on various factors such as network conditions, hardware issues, or software updates.
+    To ensure that the SDK has an up-to-date view of the network topology, it periodically retrieves information about the available nodes and their respective performance metrics. This information is used to determine which nodes to send requests to and to adjust the load balancing and failover strategies accordingly.
+
+-   `setAutoValidateChecksums` - Configure automatic checksum validation
+
+### Logging
+
+-   `setLogger` - Configure client logger. Setting a logger can give you additional information for debugging processes. This is an example of how you can use this fucntionality.
+
+```javascript
+const infoLogger = new Logger(LogLevel.Info);
+client.setLogger(infoLogger);
+```
+
+There are different LogLevels: `LogLevel.Info`, ``LogLevel.Silent`, `LogLeve.Trace`, `LogLevel.Debug`, `LogLevel.Warn`, `LogLevel.Error`, `LogLevel.Fatal`.
+
+### Default values:
+
+| Key Name                          | Default Value                |
```

Comment:
I would actually move this to the top right below the table with operator.
As a dev I want to quickly know what all my options are with summaries.
Then I can read further down to dig into details